addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -8,16 +8,16 @@
 class AccountBankStatement(models.Model):
     _inherit = 'account.bank.statement'
 
-    def unlink(self):
+    @api.ondelete(at_uninstall=True)
```

Comment:
unsure, but in theory this will raise a non-blocking error during uninstall *AND* leave the data intact in the database which might be what we want for a "certification" module.

this comment applies to almost all unlink overrides within `l10n_fr_pos_cert`

Reply:
Indeed, this does not prevent module uninstallation, but in that case the given bank statements will not be deleted.  The user can actually delete those bank statements after module uninstallation.